Formula/gjstest.rb

```diff
@@ -1,11 +1,29 @@
 class Gjstest < Formula
   desc "Fast JavaScript unit testing framework that runs on the V8 engine"
   homepage "https://github.com/google/gjstest"
-  url "https://github.com/google/gjstest/archive/v1.0.2.tar.gz"
-  sha256 "7bf0de1c4b880b771a733c9a5ce07c71b93f073e6acda09bec7e400c91c2057c"
   revision 13
```

Comment:
Ups, I've missed to bump the revision here.

Reply:
Yeah that'll need doing, sadly.

Reply:
Yeah I know. I've already fixed it locally, but didn't want to trigger another 1h long build before getting more review comments to address, because the CI has quite some backlog already.

Reply:
Aye, CI is essentially an active disaster zone at the moment thanks to one of the workspaces falling on its face.